mpls/draft-ietf-detnet-mpls.xml

```diff
@@ -885,6 +889,8 @@ DFx = DetNet member flow x over a TE LSP
     description of the Associated Channel mechanism, and to the
     DetNet work on OAM for more information DetNet OAM.
    </t>
+   <t> Additional considerations on DetNet specific OAM are further study.
+   </t>

```

Comment:
"DetNet specific" are subjects for further study.

Reply:
are you still planning on making this change?